openmrs/apps/customDisplayControl/js/customControl.js

```diff
@@ -195,6 +198,10 @@ angular.module('bahmni.common.displaycontrol.custom')
             var jitsiMeetingId = $scope.upcomingAppointmentsUUIDs[appointmentIndex];
             appService.setTeleConsultationVars(jitsiMeetingId, true);
         };
+        $scope.showJoinTeleconsultationOption = function (appointmentIndex) {
```

Comment:
Please refrain from hard-coding texts. 'Scheduled' - please pull it from config.

Reply:
As we don't have any constant file currently, so having it here.
But point taken to have a constant file going ahead.